apps/settings/elements/wifi.html

```diff
@@ -47,6 +47,15 @@ <h2 data-l10n-id="availableNetworks"> Available Networks </h2>
         <h2 data-l10n-id="advancedSettings"> Advanced Settings </h2>
       </header>
       <ul>
+        <li id="wifi-sleepMode" class="hint">
```

Comment:
It is discouraged to use id in settings app to reduce the naming conflicts as all settings panel live in the same dom tree. Please use a class name instead.